requirements.txt

```diff
@@ -1,2 +1,5 @@
+iniconfig==2.0.0
+packaging==24.1
+pluggy==1.5.0
```

Comment:
Are these packages really needed at this point?

Reply:
I believe so, this is what `pip freeze` spits out after installing `pytest`. I found a way to separate out development dependencies, but I'm not 100% sure I love the implementation (commit incoming).

Reply:
Ah I see, if they are just dependencies of pytest I thinks it's fine 🙂

Reply:
Moved them to a separate file `requirements-dev.txt`, which would install the content of `requirements.txt` plus these. What I dislike about the implementation is that you can no longer use `pip freeze > requirements.txt` as that would give you _all_ dependencies. And the same goes for the dev deps, there's now a manual selection of which dependency goes in which file.

If you're fine with keeping all dependencies in one `requirements.txt` I'm in favor of that. Let me know if I should undo the split?

Reply:
Nice! I'd prefer keeping the two requirements files. It keeps the environment minimal. I don't expect that we'll be adding many dependencies going forward since the tool meant to be rather lightweight.